AdyenTwint/TwintComponent.swift

```diff
@@ -35,23 +123,49 @@ public final class TwintComponent: PaymentComponent {
     /// - Parameter configuration: The configuration for the component.
     public init(paymentMethod: TwintPaymentMethod,
                 context: AdyenContext,
-                configuration: Configuration = .init()) {
+                configuration: TwintComponentConfiguration = .init()) {
```

Comment:
Configuration is a typealias

```suggestion
                configuration: Configuration = .init()) {
```

Reply:
Changed!